app/src/nxt/res/layout/fragment_nxt_firmware.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hey, I fixed the orientation of them and the following picture shows the updated result. 
![screenshot_20161002-132806](https://cloud.githubusercontent.com/assets/2739915/19018611/4c382744-88a4-11e6-965b-cf08b4618dd8.png)